bukkit/src/main/java/me/leoko/advancedban/bukkit/BukkitMethods.java

```diff
@@ -186,7 +186,7 @@ public boolean hasPerms(Object player, String perms) {
     @Override
     public Permissionable getOfflinePermissionPlayer(String name) {
         OfflinePlayer player = Bukkit.getOfflinePlayer(name);
-        if(permissionVault == null || player == null || !player.hasPlayedBefore())
+        if (permissionVault == null || player == null || !player.hasPlayedBefore())
```

Comment:
Normally I wouldn't make formatting changes but it's the only place in the file where the auto-formatter made changes.